querybook/webapp/components/ListDataDoc/components/LastRecord.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+
+export const LastRecord: React.FunctionComponent<{
+    value: { created_at: number; updated_at: number };
```

Comment:
need a better name for value and what the type is, are you passing run record here?